### PR TITLE
Added doc strings for AdminDao class and its functions

### LINKED ISSUE #153: Create doc strings for AdminDao class and its functions
## Description
As a developer,
I need to have in code documentation,
so that I can understand better the code.

Related to #149 

We need to have **doc strings for AdminDao class and its functions**. You can find this file [app/api/dao/admin.py](https://github.com/systers/mentorship-backend/blob/develop/app/api/dao/admin.py).
You can follow these conventions: [style guide for doc strings in python by Google](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).

### Update [Required]
- [ ] Create doc strings for AdminDao class and its functions (file: app/api/dao/admin.py).

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by sarnava1**
### Description
Added doc strings for AdminDao class and its functions
This PR was created after there were some errors during the squashing of commits in the previous PR which had tried to do the same work. The link is https://github.com/systers/mentorship-backend/pull/157.

Fixes #153 

### Type of Change:

- Quality Assurance
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by sarnava1** [comment]
@isabelcosta I think this PR is fine now. Please review it.

**Comment by isabelcosta** [comment]
When you create a new PR like this related to another PR or issue relate them in some way to keep track of the history. Please update pr description with a link to the other PR @sarnava1

**Comment by sarnava1** [comment]
@isabelcosta done.

**Comment by sarnava1** [comment]
Please review @isabelcosta.

File: app/api/dao/admin.py [hunk 1]
```diff
@@ -2,9 +2,20 @@
 
 
 class AdminDAO:
-
+    """Data Access Object for Admin functionalities."""
```
Comment (isabelcosta) [inline app/api/dao/admin.py:5]: Space below this line please.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@sarnava1 Left one change request.

Also the commit message does not have a space between "docs" and ":" so:
Before:
```
docs : added doc strings for AdminDao class and its functions
```

After:
```
docs: added doc strings for AdminDao class and its functions
```

If you get any problem with git or terminal stuff, tell me.
I'll be more than happy to help you :)

File: app/api/dao/admin.py (hunk 1 above)
Comment (isabelcosta) [reply app/api/dao/admin.py:5]: one line below only

**Comment by sarnava1** [comment]
@isabelcosta please review the PR. I will squash the commits if you give the green signal.

**Review by isabelcosta — DISMISSED** [review]
Green signal given!

**Comment by isabelcosta** [comment]
@sarnava1 remember about the comment I made on the commit message. Your 2° commit didn't change

**Comment by sarnava1** [comment]
@isabelcosta Actually I could not understand what I should do!! Initially I gave "docs: added ...." and now I gave "docs : added ..." 

Which should I finally give?

**Comment by isabelcosta** [comment]
First option you mentioned @sarnava1

**Comment by sarnava1** [comment]
Ok @isabelcosta

**Comment by sarnava1** [comment]
@isabelcosta I am getting this error:

error_1

**Comment by isabelcosta** [comment]
@sarnava1 I'm on my phone now, can you share that on slack please. I'm sure someone will help you out!

**Comment by sarnava1** [comment]
@isabelcosta sure!

**Comment by isabelcosta** [comment]
Uh wait. Always check if you can fix Codacy warnings. You have one error pending @sarnava1 fix that please

**Comment by sarnava1** [comment]
@isabelcosta the codacy warning is because of the blank line I inserted as you asked me to do. Initially I did not put that blank line because it was giving the codacy errors.

**Comment by isabelcosta** [comment]
@sarnava1 I verified now that the trailing whitespace problem isn't related to the new line but with the space characters you have in that blank line:

image

highlighted in blue is the trailing whitespace. You can add a blank line without spaces in that line. Do you understand @sarnava1?

**Comment by sarnava1** [comment]
Ok @isabelcosta I am trying it out.

**Comment by sarnava1** [comment]
@isabelcosta the checks have passed!!

**Comment by isabelcosta** [comment]
Yey! @sarnava1 you learned something here ;)

**Comment by isabelcosta** [comment]
Ok so now you can try to squash the commits :)

**Comment by sarnava1** [comment]
@isabelcosta I learnt a lot here!! I will try to squash the commits.

**Comment by sarnava1** [comment]
@isabelcosta I am getting the same error again!! I have posted it in the slack channel too!

**Review by isabelcosta — APPROVED** [review]
Good job @sarnava1 ! Thank you for your patient :) I'm glad you got to solve your git problem!

**Pull request merged**